ipld-importer/printing-dagservice.go

```diff
@@ -0,0 +1,63 @@
+package importer
```

Comment:
same comments as for the other dag service. you might want to use a `logger` rather than printLn. Or maybe in the future it is just worth to log as debug messages in the other dagService and simply delete this (I understand this is just for easier testing now).